build-buildscript.gradle

```diff
@@ -0,0 +1,36 @@
+dependencies {
+	classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "6.1.194"
+	classpath group: "com.liferay", name: "com.liferay.gradle.plugins.maven.plugin.builder", version: "1.2.6"
+	classpath group: "de.undercouch", name: "gradle-download-task", version: "3.3.0"
+	classpath group: "gradle.plugin.org.ysb33r.gradle", name: "gradletest", version: "2.0"
+	classpath group: "xalan", name: "xalan", version: "2.7.2"
+
+	if (Boolean.getBoolean("gradle.build.scan.enabled")) {
+		classpath group: "com.gradle", name: "gradle-enterprise-gradle-plugin", version: "3.1.1"
```

Comment:
@ling-alan-huang Why are we adding build scan here? 

We already have build scans enabled https://github.com/liferay/liferay-blade-cli/blob/master/build.gradle#L5

Reply:
@lawrence-lee, this is the first runable version, I am looking into it and will remove something we don't need.